services/stickers/helpers.js

```diff
@@ -170,35 +190,39 @@ export async function notifyVoters(data, action, event) {
  *
  * sends message to all admins
  */
-export async function notifyAdmins(data, action, event) {
-  let voters;
+export async function notifyAdmins(data, action, event, roomID) {
+  let admins;
   try {
     const command = new QueryCommand({
       IndexName: "role",
       ExpressionAttributeNames: {
-        "#role": "role"
+        "#role": "role",
+        "#roomID": "roomID"
       },
       ExpressionAttributeValues: {
-        ":role": "admin"
+        ":role": "admin",
+        ":roomID": roomID
       },
       KeyConditionExpression: "#role = :role",
+      FilterExpression: "#roomID = :roomID",
       ProjectionExpression: "connectionID",
       TableName: SOCKETS_TABLE + (process.env.ENVIRONMENT || "")
     });
     const response = await docClient.send(command);
-    voters = response.Items;
+    admins = response.Items;
   } catch (error) {
     let errResponse = db.dynamoErrorResponse(error);
     console.error(errResponse);
   }
 
-  for (let i = 0; i < voters.length; i++) {
+  // should only ever be length one because we have one presenter per room
+  for (let i = 0; i < admins.length; i++) {
     await sendMessage(
       {
```

Comment:
If any presenter devices dont gracefully disconnect their entry will be left in the DB which is why i left the loop